bbb-convert-stage-5.sh

```diff
@@ -215,6 +213,9 @@ install_files() {
   sudo ln -fs /sbin/fw_printenv $rootfs_dir/usr/bin/fw_printenv
   sudo ln -fs /sbin/fw_setenv $rootfs_dir/usr/bin/fw_setenv
 
+  #Create links for grub
+  sudo ln -sf /boot/dtbs/$linux_version/am335x-boneblack.dtb $rootfs_dir/boot/dtbs/am335x-boneblack.dtb
+  sudo ln -sf /boot/vmlinuz-$linux_version  $rootfs_dir/boot/vmlinuz
```

Comment:
Please use generic names here. e.g:

/boot/dtb
/boot/kernel

I would expect that we will have this problem on more boards running Debian, and no point in hard-coding it to "bbb".

Reply:
And the kernel image format will differ between boards.

Reply:
done